.github/workflows/matrix.js

```diff
@@ -0,0 +1,35 @@
+let { MatrixBuilder } = require('./matrix_builder');
+const matrix = new MatrixBuilder();
+
+// Add axes for the matrix
+matrix.addAxis({
+    name: 'os',
+    title: x => x.replace('-latest', ''),
+    values: [
+        'ubuntu-latest',
+        'windows-latest'
+    ]
+});
+matrix.addAxis({
```

Comment:
You also need to add an axis for distribution. I suggest you use:
'corretto',
		'liberica',
		'microsoft',
		'temurin',
		'zulu',